public/index.html

```diff
@@ -194,6 +194,12 @@
                 href="https://blog.statechannels.org"
                 >blog.</a
               >
+              <a
+                class="nav__right__item"
+                target="_blank"
+                href="mailto:contact@magmo.com"
+                >contact.</a
```

Comment:
I'm not married to this change, but even after glancing at the change set I was still surprised by this being a mailto link when I loaded the preview. As is, it *looks like* a typical link and I expected it to go somewhere.

Maybe the word `email` instead of `contact` would also work (and fit in better with the rest of the menu).

```suggestion
                >contact@magmo.com</a
```